Makefile

```diff
@@ -314,7 +313,7 @@ operator-sdk:
 	# Download operator-sdk locally if does not exist
 	if [ ! -f $(OPERATOR_SDK) ]; then \
 		mkdir -p bin ;\
-		curl -Lo $(OPERATOR_SDK) https://github.com/operator-framework/operator-sdk/releases/download/v1.23.0/operator-sdk_$(shell go env GOOS)_$(shell go env GOARCH) ; \
+		curl -Lo $(OPERATOR_SDK) https://github.com/operator-framework/operator-sdk/releases/download/v1.34.2/operator-sdk_$(shell go env GOOS)_$(shell go env GOARCH) ; \
```

Comment:
I got lost in slack thread, why we are updating here?

if it because of new flag, should be safer to update to lowest release that contains it? I think it is https://github.com/operator-framework/operator-sdk/releases/tag/v1.24.1

and if we do update, I think at least the annotations telling which operator-sdk is used by OADP need to be updated. I have similar changes here https://github.com/openshift/oadp-operator/pull/1250

Reply:
So are we  good with this as-is to get this merged ASAP to have a working oadp-1.4 branch? Follow-on PRs to refine it as needed.

Reply:
Yes, Rayford indicated that the operator-sdk is NOT used in d.s. builds anyway.  This is dev only.  Clear to proceed.

Reply:
we need operator-sdk to generate `bundle` folder (`make bundle` command). So I think it is not only for development

I will make some tests, but maybe is ok to update version (and if needed changes, can be a follow up PR)

Reply:
Generating bundle is a dev activity. Unless we specifically ask or they feel adventurous, `make bundle` is only something our team touches.

Reply:
my fear is the new version of operator-sdk will generate new bundle, changing delivered yamls to users. BUT need to test if that really happens

Reply:
> changing delivered yamls to users

Not our downstream builds no. Upstream builds are not published in operatorhub so we should be good.

Reply:
Usually the (upstream) changes are just CSV annotations/labels